### DISCUSSION TITLE
I converted my personal account to an organization account and now I can't log in and I specified the owner of the wrong account.

### DISCUSSION THREAD

**Discussion by kokhlikyan** [community/community#139461] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I accidentally converted my personal GitHub account into an organization and now I am unable to log in or manage it as a personal account. I would like to revert my account back to a personal account, if possible.

Could you please assist me in restoring my account or provide guidance on how to resolve this issue? I appreciate your support in this matter.

Thank you for your help!

**Comment by ebndev** [comment]
Hi @norayr95,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
